src/test/java/github/weichware10/util/db/TrialsTest.java

```diff
@@ -0,0 +1,309 @@
+package github.weichware10.util.db;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+import github.weichware10.util.Logger;
+import github.weichware10.util.Enums.ToolType;
+import github.weichware10.util.config.CodeChartsConfiguration;
+import github.weichware10.util.config.Configuration;
+import github.weichware10.util.config.ZoomMapsConfiguration;
+import github.weichware10.util.data.DataPoint;
+import github.weichware10.util.data.TrialData;
+import io.github.cdimascio.dotenv.Dotenv;
+
+import java.util.ArrayList;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'java.util.ArrayList'